src/main/resources/META-INF/plugin.xml

```diff
@@ -119,7 +119,7 @@
 
             <action id="GitMachete.FastForwardParentToMatchCurrentBranchAction"
                     class="com.virtuslab.gitmachete.frontend.actions.toolbar.FastForwardParentToMatchCurrentBranchAction"
-                    icon="AllIcons.Vcs.Merge">
+                    icon="AllIcons.Diff.ApplyNotConflictsLeft">
```

Comment:
There's a problem with hijacking icons like that: they're subject to change b/w Intellij versions, and at some point they might be changed so that they still will be adequate for their original purpose, but no longer match our use case. Maybe instead copy the original icon to our project?... TBD how that works with the IntelliJ's Apache license.

Reply:
This comment likely already applies to a few other off-label/hijacked uses of icons in the plugin as well.

Reply:
For now, I would like to keep things like this. So let's use JB icons tath (presently) just looks (close to) accurate.

Your concerns reasonable (and actually they are not new in the scope of our plugin) and custom icon set deserve their own issue definitely (#326). Assuming that icons creation will take some time we shall carefully think through our targets, requirements, and options.

Reply:
Hmm there are in fact 2 separate issues here:
1. whether to provide our own icons or rely or IntelliJ's icons for certain actions
1. if we rely on IntelliJ's icons, then make sure that our UX isn't susceptible to IntelliJ changing the look of those icons b/w versions

Reply:
(2) is relatively easy to do, we can e.g. just copy the icons directly into our project

Reply:
You are right. I will take care of the second issue on separate PR

Reply:
Actually this seems to be a good point to do this since we have seen `the frontendIcons module` created.

Reply:
PR #328